src/components/mdx-img/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from 'react'
+import { MdxImgProps } from './types'
+import classNames from 'classnames'
+import s from './mdx-img.module.css'
+
+function MdxImg({
```

Comment:
Can you tell me more about your reasoning behind this name? I think `Image` would be an appropriate name, because we already have `Heading` and `Text` for example that are used in MDX content on docs views.

Reply:
I think it's interesting to have a way to define which components are used in MDX. Presumably there will be other views on DevDot where we could use a generic `Image` component that needs to be different from this one. Maybe just a comment in the file suffices for now.

Reply:
The reasoning was that we likely don't want to use this component outside of MDX contexts. The component is intended solely for use as an MDX custom component (I think in JSX contexts, the use of `MdxImage` and `ImageConfig` would be an odd and opaque way to get the desired markup and styles). As well, I was a little worried about colliding with the `next/image` import name (often `Image`).

Can definitely see how a comment might be more useful though. I've renamed `MdxImage` to `Image`. I think another route we could go here, if capturing that intent of "some components should be used exclusively in MDX, not JSX" resonates, is to rename `ImageConfig` to `MdxImageConfig`.

Reply:
To clarify, my comment was only asking for context, not asking for a rename. If we want this component to only be used in MDX, I think that's fine and we don't have to use a more general name of `Image`. 💖

Reply:
Ah cool, yeah I don't feel strongly so figured I might as well go with `Image` as you suggested - I dig the consistency with `Heading` and `Text`, and like that it now sits alongside `ImageConfig`. The `next/image` thing is pretty minor; i don't think there's much of a genuine risk for confusion, the name overlap just jumped out to me (and we can always adjust later, eg if we were to try to use `next/image` in MDX). And I guess another way to make the MDX intent clear could be a `components/mdx` folder or something (similar to `author-primitives` i guess? and pending swingset support for nested folders).

Happy to go with either `MdxImg` or `Image`, in the absence of anyone having a strong preference I'll leave as is with `ImageConfig` & `Image` ☺️